coops/sentinels.tex

```diff
@@ -124,35 +123,33 @@ \subsection*{\MakeUppercase{Combat with\\Enemy Armies}}
 
 \begin{itemize}
   \item Fighting enemy armies over more than one Combat Round does not cost Movement Points.
-  \item During a battle with an enemy army, a Hero can retreat whenever a friendly Unit is about to Activate.
-    A retreating Main Hero loses all remaining MPs and is returned to an allied Town or Settlement of their choice, keeping all remaining Units.
+  \item During a battle with an enemy army, a Hero can retreat whenever a friendly Unit is about to Activate. A retreating Main Hero loses all remaining MPs and is returned to an allied Town or Settlement of their choice, keeping all remaining Units.
   \item Retreating Secondary Heroes are removed from the game, but their remaining Units are kept.
   \item Killed enemy Units do not respawn.
 \end{itemize}
 
+\begin{tikzpicture}[overlay, anchor=north west]
+  \node {\includegraphics[width=\columnwidth, keepaspectratio]{\art/berserk.png}};
+\end{tikzpicture}
+\vspace*{\fill}\columnbreak
+
 \subsection*{\MakeUppercase{Boss Army}}
 
 \begin{itemize}
   \item In Round 10, a Boss Army spawns. It is different from previous armies because it is Reinforced from a fixed pool of Neutral Units (Reinforcement pool = number in brackets in the table below).
   \item At the beginning of the \nth{2} Combat Round and every following Round, 2 Units Reinforce the remaining army from the shuffled reinforcement pool (up to a maximum of 5 Units) to continue the fight until either the player retreats or every Neutral Unit from the Reinforcement pool is killed. \textit{If the number of enemy Units at the beginning of a Round is 1 or lower, draw up to a total of 4 Units (i.e., there are always 4 or 5 enemy Units on the Board after Reinforcing, if available).}
   \item Place the Reinforcing Units on the enemy base line, starting from the left with the lowest Initiative (\svg{unit_ranged} Units first). If necessary, continue on the next line in the same order.
+  \vspace*{\fill}\columnbreak
   \item When a player retreats, the enemy army draws up to the ``minimum'' of 4 Units and takes these as starting Units to the next fight.
 \end{itemize}
 
-\columnbreak
-
-\vspace*{\fill}
-
-\end{multicols}
-
-\begin{tikzpicture}[overlay]
-  \centering
-  \node at (14, 2) {\includegraphics[width=0.4\paperwidth]{\maps/sentinels.png}};
+\begin{tikzpicture}[overlay, anchor=north west]
```

Comment:
Same here, since it lands in a column like the text does, make it simple includegraphics with vfills surronding it, so that it's spaced equally between the text and the table below.